astronomer/airflow/version_check/update_checks.py

```diff
@@ -230,9 +232,10 @@ def parse_version(rel):
             ver = version.parse(release['version'])
             if release['channel'] in ['alpha', 'beta']:  # ignore alpha & beta releases
                 continue
-            if ver <= current_version:
+            if ver < current_version:
                 self.log.debug(
-                    "Got to a release (%s) that is older than the running version (%s) -- stopping looking for more",
+                    "Got to a release (%s) that is older or equal to the running version (%s) -- stopping looking "
+                    "for more",
```

Comment:
```suggestion
                    "Got to a release (%s) that is older than the running version (%s) -- stopping looking for more",
```

The original log message is right now - we got to an older version.